### DISCUSSION TITLE
how to make site https automatic

### DISCUSSION THREAD

**Discussion by 99839** [leokhoa/laragon#756] [General]
when i create a new site, and open it in laragon menu, it default http. not https. but when i open https manually, it is working. please help me to how to setting.

**Comment by 99839** [comment]
![1.png](https://s2.loli.net/2024/04/14/7ymzKJcf4FLrude.png)
![2.png](https://s2.loli.net/2024/04/14/3y7ZifdNa5DH1B8.png)
![3.png](https://s2.loli.net/2024/04/14/5wePvX13qLoWInU.png)
![4.png](https://s2.loli.net/2024/04/14/iw6o1ct4dKZu29S.png)

**Comment by Pen-y-Fan** [comment]
I don't use WordPress, so I can't confirm myself. Take a look at this [SO post](https://stackoverflow.com/questions/32049820/use-htaccess-to-redirect-http-to-https)

**Reply by 99839** [reply]
thank you very much for your reply

**Comment by dustinatx** [comment]
For WordPress, edit wp-config.php. Remove:

```
if ( !defined('WP_CLI') ) {
    define( 'WP_SITEURL', $_SERVER['REQUEST_SCHEME'] . '://' . $_SERVER['HTTP_HOST'] );
    define( 'WP_HOME',    $_SERVER['REQUEST_SCHEME'] . '://' . $_SERVER['HTTP_HOST'] );
}
```

Now you can edit WordPress Dashboard > Settings > General. Add the https domain as the WordPress Address and Site Address.

**Reply by 99839** [reply]
thanks, bro, it is working for edit https.  

i find .htaccess  to redirect http to https, and can not find method when use laragon menu to open site with https

**Comment by TheOldMan2000** [comment]
Hi @dustinatx @99839 

I had the same problem. Thank you for the suggestion to comment out those lines, it got my local Laragon WordPress site up and running perfectly behind SSL.

Saved me a lot of time!

**Comment by calcgenie** [comment]
try to change it in the wordpress settings page for the best and make a look at the htaccess also. i have used this code to fix it. 

RewriteEngine On
RewriteCond %{SERVER_PORT} 80
RewriteRule ^(.*)$ https://www.yourdomain.com/$1 [R,L]

It worked very well in my case. after some i have some other issues then i switched my website into wix for these websites called https://www.splitle.com, https://www.ibovistaffing.com and https://www.ibovi.com/ 


All Links

​​https://www.splitle.com/

​ https://www.splitle.com/account-finance-industry-candidate

​ https://www.splitle.com/accounting-finance

https://www.splitle.com/accounting-finance-industry-recruiter​

https://www.splitle.com/administrative-industry ​
 https://www.splitle.com/administrative-industry-candidate 
​ https://www.splitle.com/administrative-industry-recruiter ​ 
https://www.splitle.com/authorised-countries ​ 
https://www.splitle.com/blog
​ https://www.splitle.com/commissions

​ https://www.splitle.com/construction-industry

​ https://www.splitle.com/construction-industry-candidates ​

https://www.splitle.com/construction-industry-recruiter

https://www.splitle.com/contract-hire

https://www.splitle.com/contract-to-hire

https://www.splitle.com/direct-hire

​ https://www.splitle.com/engineering-industry

​ https://www.splitle.com/engineering-industry-candidate

https://www.splitle.com/engineering-industry-recruiter

https://www.splitle.com/faq

https://www.splitle.com/features ​

https://www.splitle.com/health-care-industry
​

https://www.splitle.com/health-care-industry-candidate

https://www.splitle.com/health-care-industry-recruiter


https://www.splitle.com/how-splitle-work

https://www.splitle.com/it-industry

https://www.splitle.com/it-industry-candidate

https://www.splitle.com/it-industry-recruiter

https://www.splitle.com/manufacturing-industry

​https://www.splitle.com/manufacturing-industry-candidate

​https://www.splitle.com/manufacturing-industry-recruiter

https://www.splitle.com/pharmaceutical-industry

https://www.splitle.com/pharmaceutical-industry-candidate

https://www.splitle.com/pharmaceutical-industry-recruiter
